scripts/laptop.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+source install/setup.bash && \
+ros2 launch gstreamer laptop_launch.py
```

Comment:
This MUST open a container first

Reply:
we have a separate script for that, might put it into one later on though